src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
+            mCurrentActivity = activity;
         }
 
         @Override
         public void onActivityResumed(Activity activity) {
             installConnectionSensor(activity);
             mEditState.add(activity);
+            mCurrentActivity = activity;
+            ConnectivityIndicator.show(activity);
```

Comment:
It's better to check for `mMixpanelInEditMode` here. `show` should show a view and it should not be it's responsibility to check whether there is a socket open or not, etc..